node-zerox/src/openAI.ts

```diff
@@ -46,6 +47,7 @@ export const getCompletion = async ({
         messages,
         model,
         temperature: 0,
+        ...llmParams,
```

Comment:
Since temperature is repeated here, lets add that as one of the llm params with a default of 0